src/index.html

```diff
@@ -4,65 +4,375 @@
+            <label for="f-option">Vote</label>
+            <div class="check"></div>
+          </li>
+
+          <li>
+            <input type="radio" id="s-option" name="selector">
+            <label for="s-option">UnVote</label>
+            <div class="check">
+              <div class="inside"></div>
+            </div>
+          </li>
+        </ul>
+
+      </div>
+      <div class="modal-body">
+        <p style="margin-top: 80px;">Partners with:</p>
+        <img id="a" style="max-width: 110px;" src="logos/lucky.png" alt="">
+        <div class="columns is-vcentered">
+            <!-- <div class="column">
+                <div class="field">
+                    <input id="delegationSwitch" type="checkbox" name="switchLarge" class="switch is-large is-rtl">
+                    <label for="delegationSwitch">I want to delegate my votes to you!</label>
+                </div>
+            </div> -->
+            <div class="column">
+                <!-- Callout explaining the rules we're following when voting on your behalf -->
+                <!-- <div class="notification is-warning is-light">
+                    Delegate votes to the Kusamarian voting collective
+            </div> -->
         </div>
-    </section>
-    <div class="is-divider"></div>
-    <section>
+    </div>
+        <div class="input-group">
+          <input value="" class="form-control" type="text" name="text-1542372332072" id="text-1542372332072"
+            required="required" placeholder="DOT Amount">
+          <label for="text-1542372332072">DOT Amount</label>
+        </div>
+
+        <div class="filters">
+          <select id="main_select_services" class="select-club-services" name="club-services">
+            <option value=".abrikosovaya" selected>7 Days <br> x1</option>
+            <option value=".bratskaya">14 Days <br> x2</option>
+            <option value=".lesi-ukrainki">28 Days<br> x3</option>
+            <option value=".lesi-ukrainki">56 Days <br> x4</option>
+            <option value=".lesi-ukrainki">112 Days <br> x5</option>
+            <option value=".lesi-ukrainki">224 Days <br> x6</option>
+          </select>
+        </div>
+        <div style="width: 100%; text-align: center;">
+          <!-- <button class="delegate-button">Vote</button> -->
+          <button id="myBtnW" class="delegate-button">Vote</button>
+        </div>
+      </div>
+    </div>
+
+  </div>
+
+  
+
+  </div>
+
+
+
+
+
+
+
+
+ 
+    
+
+    <!-- <section>
         <div class="container">
             <div class="columns is-centered">
                 <div class="column is-narrow">
                     <button class="button is-success is-large">Submit transaction</button>
                 </div>
             </div>
         </div>
-    </section>
+    </section> -->
+
+    <script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js'></script>
     <script type="module">
         import '/src/app.js';
+
+
+        function preLoad() {
```

Comment:
I'd rather keep JS outside of the HTML markup, if possible.
You can either extend the `/src/app.js` or add your own, like `/src/frontend.js` — both will work.
It will be inlined by `parcel` anyway, so no need to worry about performance impact of having separate files blocking the http pipeline.